front/pages/api/stripe/webhook.ts

```diff
@@ -69,7 +80,7 @@ async function handler(
           },
         });
       }
-
+      const stripeSubscription = event.data.object as Stripe.Subscription;
```

Comment:
Not sure we want this out of the switch, depending on the event it's not necessarily a Stripe.Subscription right?

Reply:
You did not want to handle this one? 🙏🏻